scenarios.md

```diff
@@ -199,6 +199,26 @@ A deployment requires a mydb image. The mydb image is routinely updated for secu
 - Multiple signatures, including signatures from multiple sources can be associated with a specific artifact.
 - Original signatures are maintained, even if the artifact is re-tagged.
 
+### Scenario 7: A mirror is compromised
+
+An attacker compromises a server that is hosting a mirror and gains access as an administrator. The attacker is able to alter any files on the mirror and access all keys stored on the mirror.
+
+**Implications of this requirement:**
+
+- There must be a way to check a signature from the original repository/developer when downloading from a mirror.
+- Clients should not trust files that are signed only by the mirror, and not by the original repository.
+- Mirrors should not have permission to edit packages and signatures.
+
+### Scenario 8: An attacker performs a MitM attack
+
+An attacker is able to sniff network traffic between the repository and client and alter packets as they are passed through the network.
+
+**Implications of this requirement:**
+
+- All downloaded files and metadata files must be validated, even if they are downloaded from a known repository.
```

Comment:
What does "validated" imply here? SSL transmission with current TLS support. Or, are you suggesting some sort of reproducible build like scope?

Reply:
This could mean TLS, signature verification, or any other technique that does not automatically trust files. I can work on a more clear way to word that